clang/lib/CIR/CodeGen/CIRGenModule.h

```diff
@@ -335,6 +335,11 @@ class CIRGenModule : public CIRGenTypeCache {
   /// in AST is always in default address space.
   LangAS getGlobalConstantAddressSpace() const;
 
+  /// Returns the address space for temporary alloca in the language. This makes
+  /// the allocated variable's address matches the expectation of AST, rather
+  /// than keep the address space of the alloca in CIR.
```

Comment:
- "allocated variable's address matches" -> "allocated variable's address space match"
- do we really "keep" the AS; isn't it rather something like "defer mapping" or so?

Reply:
Nice catch, thanks~

> do we really "keep" the AS; isn't it rather something like "defer mapping" or so?

Sorry for my dumbness, could you please elaborate the "defer mapping" approach?

Reply:
Sorry, just a minor nit, I meant that by using the language's AS in CIR, we *defer the mapping* to the target-specific AS that is used for the `alloca` instruction later in the pipeline. We don't *keep* it because that `alloca` has not been created yet.

Reply:
Oh I got it. I've forgotten what I wrote in the comment😂Surely will fix it.